### PR TITLE
DO-NOT-MERGE: experiment with timestamps within the buildah step

### PR DESCRIPTION
**Pull request by gabemontero**

### CODE REVIEW

**Comment by gabemontero** [comment]
bummer ubi9 does not have `moreutils`  .... will just use `date` I suppose

File: task/buildah/0.1/buildah.yaml [hunk 1]
```diff
@@ -144,7 +144,7 @@ spec:
       fi
 
       if [ -n "${PREFETCH_INPUT}" ]; then
-        mv cachi2 /tmp/
+        mv cachi2 /tmp/
```
Comment (MartinBasti) [inline task/buildah/0.1/buildah.yaml:147]: Please remove trailing whitespaces

**Comment by gabemontero** [comment]
we are going with https://github.com/redhat-appstudio/build-definitions/pull/443 from @jhutar instead

**Pull request closed without merging**